models/fakeUploads.js

```diff
@@ -0,0 +1,89 @@
+import faker from 'faker';
+import uuid from 'uuid/v1';
+
+const fakeUploads = [
+    {
+        avatar: faker.image.avatar(),
+        description: faker.lorem.sentences(),
+        id: uuid(),
+        image: faker.image.avatar(),
+        reactions: {
+            dislike: {
+                count: 0,
+                users: {},
+            },
+            like: {
+                count: 0,
+                users: {},
+            },
+            maybe: {
+                count: 0,
+                users: {},
+            },
+        },
+        user: {
+            image: faker.image.avatar(),
+            name: faker.name.findName(),
+            userId: uuid(),
+        },
```

Comment:
what is the reasoning behind attaching a `user` object to each upload object?

Reply:
I created a separate file for it later. It was a pattern I was supposed to reverse.

Reply:
You can user this partner for now, but what it have to be is `uploader` not user. The user is the person that is logged in but not necessary always the person that is the uploader (given that they will be looking at a feed of uploads mainly from their friends). What you should do is add a fake user in your redux initial state. And use that to drive any logic where you need the user `id (not userId), name or image`.